src/plugins/telemetry/doca/doca_exporter.cpp

```
@@ -0,0 +1,256 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "doca_exporter.h"
+#include "common/configuration.h"
+#include "common/nixl_log.h"
+
+#include <doca_error.h>
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)